application/helpers/update/updates/Update_626.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+use CException;
+
+class Update_626 extends DatabaseUpdateBase
+{
+    /**
+     * @inheritDoc
+     * @throws CException
+     */
+    public function up(): void
+    {
+        $boxes = $this->db->createCommand()
+            ->select("*")
+            ->from("{{boxes}}")
+            ->order('position ASC')
+            ->queryAll();
+
+        if (!empty($boxes)) {
+            if ($boxes[array_key_first($boxes)]['position'] === 1) {
+                // we increase the position of all boxes by 1
+                foreach ($boxes as $box) {
+                    $this->db->createCommand()->update(
+                        '{{boxes}}',
+                        ['position' => $box['position'] + 1],
+                        "id = '{$box['id']}'"
+                    );
+                }
+            }
+
+            foreach ($boxes as $box) {
```

Comment:
Tested by setting DBVersion to 624

![image](https://github.com/user-attachments/assets/ef39c8f5-0dc8-4730-841d-ce165d08a816)

The criteria self-contradicts. `$box['url']` cannot be `'dashboard/view'` and `'surveyAdministration/listsurveys'` at the same time. Either use `||` instead or `in_array`.

Reply:
@mohabmes looks good

![image](https://github.com/user-attachments/assets/8931fb6a-1662-4edd-97cc-f9756f3732a6)